app/utils/fletteTilTekst.ts

```diff
@@ -13,5 +13,10 @@ export const flettefeltTilTekst = (
         throw Error('Flettefeltet søkernavn ikke sendt med');
       }
       return flettefelter.søkerNavn;
+    case ESanityFlettefeltverdi.INNSENDT_TID:
+      if (!flettefelter?.innsendtTid) {
+        throw Error('Flettefeltet innsendt tid ikke sendt med');
+      }
+      return flettefelter.innsendtTid;
```

Comment:
Her burde det kanskje ikke komme så langt at flettefeltet ikke er sendt med før det feiler?

Reply:
Det er godt mulig strukturen for flettefeltene kan forbedres, både med dette og kommentaren under - strukturen som den er nå er satt opp akkurat som i [ks-søknad](https://github.com/navikt/familie-ks-soknad/blob/master/src/frontend/context/AppContext.ts#L16), ettersom vi ikke hadde noe annen erfaring å basere oss på da vi satte opp første flettefelt. Setter opp et eget [trellokort](https://trello.com/c/4G75w7zc/120-vurdere-restrukturering-av-flettefelt) for å vurdere hvorvidt en restrukturering kan være lurt 🤓